tooltool.py

```diff
@@ -755,6 +756,24 @@ def upload(package, user, host, path):
     cmd2 = "rsync  %s/*.txt %s@%s:%s --progress" % (package, user, host, path)
     cmd3 = "rsync  %s/*.tt %s@%s:%s --progress" % (package, user, host, path)
 
+    print textwrap.dedent("""\
+        **SECURITY WARNING**
+
+        Uploading files to tooltool is a risky operation.  Please consider:
+
+          * Is this data PUBLIC or INTERNAL?  If not don't upload it!
+            INTERNAL data is non-public data that is not secret, e.g., non-redistributable binaries.
```

Comment:
"data that is not intended for public distribution, but that is **not sensitive**, e.g. non-redistributable binaries" (maybe clearer)